.devfile.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+schemaVersion: 2.1.0
+metadata:
+  name: devfile-api
+attributes:
+  controller.devfile.io/storage-type: ephemeral
+components:
+  - name: devtools
+    container:
+      image: "quay.io/devfile/universal-developer-image:ubi8-latest"
```

Comment:
Minor (?) concern: this image comes with `go 1.17.7` by default at the moment.

Reply:
I mean: UDI should be updated with 1.18.